roles/etcd/tasks/main.yml

```diff
@@ -120,7 +120,9 @@
     enabled: yes
   register: start_result
 
-- include: etcdctl.yml
+- include_role:
+    name: etcd_common
+    tasks_from: etcdctl.yml
```

Comment:
Why not just make etcd_common/tasks/main.yml call etcd_common/tasks/etcdctl.yml and drop this from etcd role which already requires etcd_common via roles/etcd/meta/main.yml?